extension/sumologicextension/extension_test.go

```diff
@@ -562,6 +562,7 @@ func TestRegisterEmptyCollectorName(t *testing.T) {
 }
 
 func TestRegisterEmptyCollectorNameForceRegistration(t *testing.T) {
+	t.SkipNow() //Skip this test for now as it is flaky
```

Comment:
```suggestion
	t.SkipNow() // Skip this test for now as it is flaky
```
Lint usually fails for me if I don't have a space here 👍